src/java/com/zimbra/qa/selenium/projects/ajax/pages/briefcase/PageBriefcase.java

```diff
@@ -1435,4 +1436,13 @@ public Boolean zVerifyPdfFilePreviewContents(String fileContent) throws HarnessE
 		return true;
 
 	}
+
+	public void Maximizebrowser(){
```

Comment:
Very minor, please change function name maximizeBrowser()<space>{. In below afterMethod, you used different case: MaximizeBrowser. Lets keep it same using camel case letter.